src/blant.c

```diff
@@ -292,6 +293,7 @@ void initialize(GRAPH* G, int k, unsigned long numSamples) {
 
     char BUF[BUFSIZ];
     _numSamples = numSamples;
+	if (k > 8) read_maps(MAX_K);
```

Comment:
Again the "read_maps" function needs a better name.

Reply:
Forgot to change

Reply:
Fixed